server/models/notifications.js

```diff
@@ -1,13 +1,51 @@
 const pool = require('../../db/index.js');
 
 module.exports = {
-  addNotification: async (values = []) => {
-    const query = `INSERT INTO notifications (date, notification, accounts_id) VALUES ($1, $2, $3)`;
+  addNotification: async (values = [], username) => {
+    console.log(values);
+
+
     try {
-      const res = await pool.query(query, values);
-      return res;
+      const query1 = `INSERT INTO notifications (date, title, guests, location, notification) VALUES ($1, $2, $3, $4, $5) RETURNING id;`
+
+      const res1 = await pool.query(query1, values);
+      console.log(res1.rows[0]);
+
+      const query2 = `INSERT INTO accounts_notifications_appointments (notifications_id, accounts_id) VALUES (${res1.rows[0].id}, (SELECT id FROM accounts WHERE username = '${username}'))`
+
+      const res2 = await pool.query(query2);
+      return res1;
     } catch (error) {
+      console.log(error);
       throw error;
     }
   },
-};
\ No newline at end of file
+
+  getNotification: async (date, email) => {
+    console.log(date,email);
```

Comment:
Don't forget to remove console.logs so other people don't have to remove it later on when they call this function

Reply:
still was using this one, but i do need to remember to get rid of them